src/components/Profile/Profile.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// import { Spinner } from 'react-spinners-css';
+import style from './Profile.module.css';
+import { displayMissionFunction } from '../../redux/mission/Mission';
+import { getRocketFromAPI } from '../../redux/Rockets/rockets';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+const Profile = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(displayMissionFunction());
+    dispatch(getRocketFromAPI());
+  }, []);
+
+  const missionList = useSelector((state) => state.missionReducer);
+  const rocketsList = useSelector((state) => state.RocketsReducer);
+
+  return (
+
+    <div className={style.myprofile}>
```

Comment:
- [ ] Great job so far 👏🏻  However your Profile page should be Composed two/three-column layout and list ONLY the rockets reserved and missions joined by the user (as per design):
- Render a list of all joined missions, This should be done using `filter();`
- Render a list of all reserved rockets, This should be done using `filter();`
- [ ] Kindly make this 👍🏻